CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ IMPROVEMENTS:
 * `azurerm_app_service` - support for configuring `app_command_line` in the `site_config` block [GH-2350]
 * `azurerm_app_service_slot` - support for configuring `app_command_line` in the `site_config` block [GH-2350]
 * `azurerm_mssql_elasticpool` - deprecated the `elastic_pool_properties` propety and moved `max_size_bytes` and `zone_redundant` to the top level [GH-2378]
+* `azurerm_key_vault` - exposed `backup` and `restore` permissions made `key_permissions` and `secret_permissions` optional [GH-2363]
```

Comment:
Could we move this one line up? we sort alphabetically here

Reply:
DOH! I knew that just forgot...